packages/pancake-uikit/src/components/Slider/styles.ts

```diff
@@ -6,7 +6,7 @@ import bunnyHeadMax from "./svg/bunnyhead-max.svg";
 import bunnyButt from "./svg/bunnybutt.svg";
 
 interface SliderLabelProps {
-  progress: number;
+  progress: string;
```

Comment:
Why is this a string?

Reply:
it can be "px" or % values. so better to use string type I think.